examples/timeseries/public.py

```diff
@@ -17,30 +18,48 @@
 # You should have received a copy of the GNU General Public License
 # along with GWpy.  If not, see <http://www.gnu.org/licenses/>.
 
-"""Plotting public LIGO data
+"""Accessing and visualising public GW detector data
 
-I would like to study the gravitational wave strain time-series around the
-time of an interesting simulated signal during the last science run (S6).
-
-These data are public, so we can load them directly from the web.
+Data from the current generation gravitational wave detectors are published
+by |GWOSCl| and freely available to the public.
```

Comment:
@duncanmmacleod is there a typo here: `GWOSCl`?

Reply:
No, this is a substitution reference define in references.rst

https://github.com/gwpy/gwpy/blob/f9ce215df631c17485a085284780c56e7aa879b2/docs/references.rst#L103-L104